Dockerfile

```diff
@@ -162,5 +156,5 @@ RUN /print-missing-deps.sh
 
 # Final build stage, create the real Docker image with ENTRYPOINT
 FROM output-image
-
-ENTRYPOINT ["/gce-pd-csi-driver"]
+COPY --from=builder /go/src/sigs.k8s.io/gcp-compute-persistent-disk-csi-driver/initialize-driver.sh /initialize-driver.sh
+ENTRYPOINT ["/initialize-driver.sh"]
```

Comment:
This is fine for now. I think in the future we'll want to remove the entrypoint so that the deployment yaml has to be explicit about whether to use the initialize script or call the driver directly.